pkg/git/git.go

```diff
@@ -288,21 +323,22 @@ func (r *Repo) latestNonPatchFinalVersions() ([]semver.Version, error) {
 	return latestVersions, nil
 }
 
-func (r *Repo) releaseBranchOrMasterRev(major, minor uint64) (rev string, err error) {
+func (r *Repo) releaseBranchOrMasterRev(major, minor uint64) (sha, rev string, err error) {
 	relBranch := fmt.Sprintf("release-%d.%d", major, minor)
-	rev, err = r.RevParse(relBranch)
+	sha, err = r.RevParse(relBranch)
 	if err == nil {
-		logrus.Infof("found release branch %s", relBranch)
-		return rev, nil
+		logrus.Debugf("found release branch %s", relBranch)
+		return sha, relBranch, nil
 	}
 
-	rev, err = r.RevParse("master")
+	const master = "master"
```

Comment:
Should we make that a package level const?

Reply:
Yes, I will adapt other places where it is used as well. :+1: